From 77be1b3e7d6ff4261114424278413a679672bb92 Mon Sep 17 00:00:00 2001
From: Alper Ak <alperyasinak1@gmail.com>
Date: Wed, 9 Jul 2025 12:10:37 +0300
Subject: [PATCH] cmake: Set minimum required version to 3.5 for CMake 4+
 compatibility

Fix:

| CMake Error in CMakeLists.txt:
|   No cmake_minimum_required command is present.  A line of code such as
|
|     cmake_minimum_required(VERSION 4.0)
|
|   should be added at the top of the file.  The version specified may be lower
|   if you wish to support older CMake versions for this project.  For more
|   information run "cmake --help-policy CMP0000".
|
|
| -- Configuring incomplete, errors occurred!

Upstream-Status: Submitted [https://github.com/thuanalg/simplelog-topic/pull/1]

Signed-off-by: Alper Ak <alperyasinak1@gmail.com>
---
 CMakeLists.txt | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index dca80f1..0fc6db0 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,3 +1,5 @@
+cmake_minimum_required(VERSION 3.5)
+
 # Add main.cpp file of the project root directory as a source file
 project(simplelog_topic_project)
 set(SOURCE_LIB_FILES src/simplelog.c )
-- 
2.43.0

